python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources.py

```diff
@@ -468,7 +468,11 @@ def get_manifest_json(self, **kwargs) -> Optional[Mapping[str, Any]]:
         return parse_manifest(project_dir, target_path)
 
 
+@deprecated
```

Comment:
Let's make this emit `deprecation_warning` as well? I don't think `@deprecated` actually does anything.